docs/dev-manuals/adr/0002-use-graphql-federation.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 3
+---
+
+# Use GraphQL Federation for API Gateway
+
+## Context and Problem Statement
+
+As we decided to use GraphQL for the API gateway, we need a technology which allows us to combine the API of different services into one unified GraphQL API.
+
+## Considered Options
+
+- GraphQL Mesh / Schema stichting
```

Comment:
Type in "stichting": stitching

Reply:
fixed with 472d10afb00ad4faf8d2e0029d1260278a1bd33f